### ISSUE TITLE
Pull all of neuroml-db.org through `nmlcc`

### ISSUE THREAD

**Issue by brenthuisman** [thorstenhater/nmlcc#21]
Assign to @llandsmeer (@thorstenhater can you make him member of the repo, shall we fork/move it under the `arbor-sim` org?).

**Comment by thorstenhater** [comment]
> Pull all of neuroml-db.org through nmlcc

There's about 1700 models in there. Not too many, but way too many to manually iterate through.

> can you make him member of the repo

Sure!

> shall we fork/move it under the arbor-sim org?).

Like we just discussed a few hours ago: as soon as we are done with the StudentConf project. We plan to co-release with the poster.

**Comment by brenthuisman** [comment]
Release can be independent of moving the project. I recall we had to ask Github for a 'move' and that it took a bit of time.

**Comment by thorstenhater** [comment]
Correct, but this is what we had discussed yesterday, including you. So, the reason for not releasing moving is simply that until we are done with that project, nmlcc will be moving too fast to be sensibly released in the proper way.

**Comment by brenthuisman** [comment]
I only remember discussing the release, not the repo.
